src/counts/embezzler.ts

```diff
@@ -0,0 +1,216 @@
+import { canAdventure, getClanLounge, itemAmount } from "kolmafia";
+import {
+  $effect,
+  $familiar,
+  $item,
+  $location,
+  $locations,
+  $monster,
+  $skill,
+  ChateauMantegna,
+  CombatLoversLocket,
+  Counter,
+  CrystalBall,
+  get,
+  have,
+  property,
+  SourceTerminal,
+  sum,
+} from "libram";
+
+const embezzler = $monster`Knob Goblin Embezzler`;
+
+export function embezzlerCount(): number {
+  return sum(embezzlerCounts, (countEntry: NameCountPair) => countEntry[1]());
+}
+
+interface CountFunction {
+  (): number;
+}
+
+type NameCountPair = [string, CountFunction];
+
+const embezzlerCounts = new Array<NameCountPair>(
```

Comment:
Why are you using the array constructor instead of an array literal here?

Reply:
Typescript gonna typescript. I was leaving a little something to launch off of in case I needed to refactor EmbezzlerFight in this PR.

Reply:
you can do `const embezzlerCounts: NameCountPair[] = [` if you want it to force you to pay attention to the types

Reply:
done!